components/TrackingComplaint/Detail/index.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div class="w-full">
+    <TrackingComplaintNoData v-show="false" />
+
+    <!-- Detail Complaint -->
+    <section class="flex flex-col gap-3">
+      <!-- Image -->
+      <div class="flex flex-col gap-3">
+        <div class="max-w-[900px] max-h-fit self-center">
+          <!-- Image Preview -->
+          <img
+            loading="eager"
+            :src="selectedImage"
+            width="900"
+            height="290"
+            class="rounded-lg mx-auto object-cover"
+            alt="gambar aduan"
+          />
+        </div>
+        <div class="w-full overflow-x-auto flex flex-row gap-1">
+          <button
+            v-for="(image, index) in images"
+            :key="'gambar' + index"
+            @click="selectedImage = image"
+          >
+            <img
+              loading="eager"
+              :src="image"
+              width="105"
+              height="65"
+              class="rounded-md max-w-[105px] max-h-[65px] object-cover"
+              :class="{
+                'opacity-50 transition-opacity duration-300':
+                  selectedImage === image,
+              }"
+              alt="gambar aduan"
+            />
+          </button>
+        </div>
+      </div>
+
+      <hr class="my-3" />
+
+      <div class="flex flex-col gap-1">
+        <h3 class="font-lato text-[12px] leading-[18px] text-gray-500">
+          Kategori Aduan
+        </h3>
+        <p class="font-lato text-[14px] leading-[21px] text-gray-900">
+          Infrastruktur / Jalan
+        </p>
+      </div>
+
+      <div class="flex flex-col gap-1">
+        <h3 class="font-lato text-[12px] leading-[18px] text-gray-500">
+          Permasalahan
+        </h3>
+        <p class="font-lato text-[14px] leading-[21px] text-gray-900">
+          Saya ingin melaporkan kondisi jalan di depan rumah saya yang saat ini
+          mengalami kerusakan berupa lubang-lubang yang cukup dalam. Hal ini
+          menjadi perhatian serius karena dapat membahayakan keselamatan
+          pengguna jalan, terutama pengendara sepeda motor dan mobil.
+        </p>
+      </div>
+
+      <div class="grid grid-cols-[1fr,60px] gap-3 justify-between">
+        <div class="flex flex-col gap-1">
+          <h3 class="font-lato text-[12px] leading-[18px] text-gray-500">
+            Lokasi Aduan
+          </h3>
+          <p class="font-lato text-[14px] leading-[21px] text-gray-900">
+            Situ aksan , Sukahaji, Kec. Babakan Ciparay, Kota Bandung, Jawa
```

Comment:
I think `Situ aksan ,` is not following Pedoman Umum Ejaan Bahasa Indonesia (PUEBI). please remove extra space before `,`

Reply:
ini masih slicing @agunghide , i not fix here